src/main/java/com/commercehub/gradle/plugin/avro/Constants.java

```diff
@@ -31,6 +31,7 @@
     static final String DEFAULT_STRING_TYPE = StringType.String.name();
     static final String DEFAULT_FIELD_VISIBILITY = FieldVisibility.PUBLIC_DEPRECATED.name();
     static final boolean DEFAULT_CREATE_SETTERS = true;
+    static final boolean DEFAULT_ENABLE_DECIMAL_LOGICAL_TYPE = true;
```

Comment:
What is your thinking for why we should have `true` be the default for this value, as opposed to `false` (which is the default in `SpecificCompiler`?  I could see a potential argument for it being more intuitive/useful for the average Java/JVM developer.  That's the main reason that we currently diverge from the upstream default for `stringType`.

Do you have any feeling for under what circumstances someone would want the `false` behavior (that is, to use `ByteBuffer`?

Having this default to `true` is an additional compatibility break, and (if we continue with it) should be called out in the change log separately.

Reply:
The only reason for keeping `ByteBuffer` for `decimal` can be supporting legacy code (existing before adding this feature)

Reply:
Good enough justification for me.  Please add a change log entry for this.

Reply:
Done